src/types/index.ts

```diff
@@ -44,6 +44,7 @@ export enum MeetingStatus {
   Succeeded,
   Failed,
   Ended,
+  Left,
```

Comment:
We also need to update the `useMeetingStatus` storybook hook documentation. It has the enum there which needs to be updated.